sdk/nodejs/cmd/run/run.ts

```diff
@@ -157,7 +157,7 @@ function throwOrPrintModuleLoadError(program: string, error: Error): void {
         }
     }
 
-    console.error("  * Yowzas, our sincere apologies, we haven't seen this before!");
+    console.error("  * We've encountered an unexpected error.");
```

Comment:
@susanev bringing the discussion from #10374 in here.  I'm not strongly opinionated on _what_ the wording actually is, so I'm happy to make the change here.  I could also see us messaging something like `Pulumi encountered an unexpected error.   Raw exception message:`.  Thoughts?